docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_gcs.rst

```diff
@@ -3,16 +3,107 @@
 How to configure a Validation Result store in GCS
 =================================================
 
-.. admonition:: Admonition from Mr. Dickens
+By default, Validations are stored in JSON format in the ``uncommitted/validations/`` subdirectory of your ``great_expectations/`` folder.  Since Validations may include examples of data (which could be sensitive or regulated) they should not be committed to a source control system.  This guide will help you configure a new storage location for Validations in a Google Cloud Storage (GCS) bucket.
```

Comment:
Good call re: the warning about sensitive / regulated data. I'll add this to the data docs as well (since they also show the validations & expectations).